build.sbt

```diff
@@ -138,7 +144,7 @@ lazy val centaurCwlRunner = project
 
 lazy val womtool = project
   .withExecutableSettings("womtool", womtoolDependencies)
-  .dependsOn(wdl)
+  .dependsOn(wdlDraft2)
```

Comment:
Is there going to be a `womtool-draft2` and a `womtool-draft3` ? Or will you add a `draft3` dependency here when womtool needs to handle both ?

Reply:
I'm hoping to have pluggable languages in `womtool` too, but that's probably a story for another PR